src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The function name `calculateRentalCost` is descriptive and clearly indicates its purpose, which is good practice.